doc/man1/flux-startlog.rst

```diff
@@ -0,0 +1,64 @@
+.. flux-help-description: Show Flux instance start and stop times
+
+================
+flux-startlog(1)
+================
+
+
+SYNOPSIS
+========
+
+**flux** **startlog**
+
+
+DESCRIPTION
+===========
+
+List the Flux instance's start and stop times, by interpreting the contents
+of the KVS ``admin.eventlog``.
+
+A ``start`` event is posted to the eventlog at startup, and a ``finish`` event
+is posted to the eventlog at finalization.  The timestamps on the two events
+are used to calculate the instance run time, which is shown in the listing
+in Flux Standard Duration format.
+
+If the current ``start`` event is not the first entry in the eventlog and
+is not immediately preceded by a ``finish`` event, then the Flux instance
```

Comment:
I had to read this several times to grok, perhaps instead: 

`If the current ``start`` event is not immediately preceded by a ``finish`` event (unless it is the first entry in the eventlog), then ...`

basically highlighting the common case and noting the uncommon case vs. listing the uncommon case first.